client/l2tp_client.c

```diff
@@ -189,6 +189,9 @@ void context_free(l2tp_context *ctx);
 static l2tp_context *main_context = NULL;
 static asyncns_t *asyncns_context = NULL;
 
+/* Force the tunnel to go over a certain interface */
+char *force_iface = 0;
```

Comment:
It would be better to store this in the `l2tp_context`, as the use of global state should really be minimized. This enables one to have multiple context bound to different interfaces within the same application (not needed at the moment, but it makes for a nicer API).